integrations/tensorflow/e2e/README.md

```diff
@@ -150,7 +150,33 @@ bazel test :e2e_tests_failing
 bazel test :e2e_tests_failing_broadcasting_test__tf__iree_vulkan
 ```
 
-## Debugging tests
+## Generated Artifacts
+
+By default, running an E2E test generates a number of compilation, debugging and
+benchmarking artifacts in `/tmp/iree/modules/`. The location of these artifacts
+can be changed via the `--artifacts_dir` flag. The generated directory structure
+for each module is as follows:
+
+```
+/tmp/iree/modules/ModuleName
+├── iree_input.mlir        # tf_input.mlir translated to IREE MLIR
+├── backend_name_1         # e.g. iree_vmla, tf or tf_ref
+│   ├── compiled.vmfb      # flatbuffer of ModuleName compiled to this backend
+│   ├── saved_model
+│   └── traces
+│       ├── trace_function
+│       │   └── log.txt    # A more detailed version of the test logs
+│       └── simple_mul
+│           └── log.txt
+├── backend_name_2
+│   └── ...
+└── tf_input.mlir          # MLIR for ModuleName in TF's input dialect
```

Comment:
can you move tf_input.mlir to the top before the directories so that it is easier to see?

Reply:
Done!